solidity/contracts/bridge/BridgeState.sol

```diff
@@ -521,8 +528,9 @@ library BridgeState {
         );
 
         require(
-            _movingFundsDustThreshold > 0,
-            "Moving funds dust threshold must be greater than zero"
+            _movingFundsDustThreshold > 0 &&
+                _movingFundsDustThreshold < self.redemptionDustThreshold,
+            "Moving funds dust threshold must be greater than zero and smaller than redemption dust threshold"
```

Comment:
```suggestion
            "Moving funds dust threshold must be greater than zero and lower than redemption dust threshold"
```

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/241/commits/f2a5d262f26294ddcbdaf4295aa33b3cc18a5418.